pre_commit_hooks/check_toml.py

```diff
@@ -13,7 +13,7 @@ def main(argv: Optional[Sequence[str]] = None) -> int:
     retval = 0
     for filename in args.filenames:
         try:
-            with open(filename) as f:
+            with open(filename, encoding='UTF-8') as f:
                 toml.load(f)
```

Comment:
Was looking at the `toml.load` implementaion and this could just be `tom.load(filename)`?

Reply:
if it works go for it

Reply:
Cool, it means we don't have to manually specify the encoding